### DISCUSSION TITLE
Bootlooping on shamiko v0.5.1

### DISCUSSION THREAD

**Discussion by Sygmstr** [LSPosed/LSPosed#2018] [General]
I tried upgrading to the new shamiko version yesterday and my phone got bootlooped. I am using a A52 5g. Has anybody else experienced this problem?

**Comment by 0xIO32** [comment]
I experienced a bootloop. A few days after install. Removing the module fixed it. (PS: At the moment, i do not care anymore, MagiskHide is sufficient for my banking apps)
